.circleci/config.yml

```diff
@@ -1058,6 +1130,71 @@ workflows:
           image_tag: '<< pipeline.parameters.pg13_version >>'
           requires: [build-13]
 
+      - upload-coverage:
+          requires:
+            - test-13_check-multi
+            - test-13_check-multi-1
+            - test-13_check-mx
+            - test-13_check-vanilla
+            - test-13_check-isolation
+            - test-13_check-operations
+            - test-13_check-follower-cluster
+            - test-13_check-columnar
+            - test-13_check-columnar-isolation
+            - test-13_tap-recovery
+            - test-13_tap-columnar-freezing
+            - test-13_check-failure
+            - test-13_check-enterprise
+            - test-13_check-enterprise-isolation
+            - test-13_check-enterprise-isolation-logicalrep-1
+            - test-13_check-enterprise-isolation-logicalrep-2
+            - test-13_check-enterprise-isolation-logicalrep-3
+            - test-13_check-enterprise-failure
+            - test-13_check-split
+            - test-14_check-multi
+            - test-14_check-multi-1
+            - test-14_check-mx
+            - test-14_check-vanilla
+            - test-14_check-isolation
+            - test-14_check-operations
+            - test-14_check-follower-cluster
+            - test-14_check-columnar
+            - test-14_check-columnar-isolation
+            - test-14_tap-recovery
+            - test-14_tap-columnar-freezing
+            - test-14_check-failure
+            - test-14_check-enterprise
+            - test-14_check-enterprise-isolation
+            - test-14_check-enterprise-isolation-logicalrep-1
+            - test-14_check-enterprise-isolation-logicalrep-2
+            - test-14_check-enterprise-isolation-logicalrep-3
+            - test-14_check-enterprise-failure
+            - test-14_check-split
+            - test-14_check-arbitrary-configs
+            - test-15_check-multi
+            - test-15_check-multi-1
+            - test-15_check-mx
+            - test-15_check-vanilla
+            - test-15_check-isolation
+            - test-15_check-operations
+            - test-15_check-follower-cluster
+            - test-15_check-columnar
+            - test-15_check-columnar-isolation
+            - test-15_tap-recovery
+            - test-15_tap-columnar-freezing
+            - test-15_check-failure
+            - test-15_check-enterprise
+            - test-15_check-enterprise-isolation
+            - test-15_check-enterprise-isolation-logicalrep-1
+            - test-15_check-enterprise-isolation-logicalrep-2
+            - test-15_check-enterprise-isolation-logicalrep-3
+            - test-15_check-enterprise-failure
+            - test-15_check-split
+            - test-15_check-arbitrary-configs
+            - test-13-14_check-pg-upgrade
+            - test-14-15_check-pg-upgrade
```

Comment:
https://app.circleci.com/pipelines/github/citusdata/citus/29243/workflows/0bf7132d-3f2e-4750-a79a-4b39dbc9f1db

We seem to be missing 2 test suites here:
 - test-13_check-citus-upgrade
 - test-13_check-arbitrary-configs

Both already seem to  have the codeclimate task in them, but they don't feed into the final upload step. Possibly missing a bit of coverage.

Which shows that maintaining this list could be problematic in the future. Looking at the [config spec](https://circleci.com/docs/configuration-reference/#requires) it doesn't seem we can require every job excluding a list (which would be the builds and style checkers etc.

We will just need to make sure that every test job is in here.